doc/developer/debugging.rst

```diff
@@ -78,14 +78,20 @@ Actually we display the running rule and why she is running (dependence update).
 Docker
 ------
 
-Edit a file in a running apache WSGI container
+Run gunicorn to reload on modified python file
 ..............................................
 
+Add the following environement variable to the geoportal container:
+
+``GUNICORN_PARAMS="-b :80 --worker-class gthread --threads 1 --workers 1 --reload"``
+
+Gaceful the running geoportl container
```

Comment:
Do a graceful restart of the running geoportal container

Reply:
Thanks, fixed :-)